hack/tools/test_module.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+
+export WORKING_DIR=test-script
+export ACCESS_KEY=1234
+export SECRET_KEY=1234
+
+kubernetesVersion=$1
+fybrikVersion=$2
+moduleVersion=$3
+module=$4
```

Comment:
Please remove `module` and add `moduleResourceVersion` as in https://github.com/fybrik/data-movement-operator/blob/master/hack/tools/test_module.sh#L19 to support module patched version.